packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -157,6 +157,10 @@ export default {
   &__emoji {
     margin-right: var(--dt-space-300);
     display: inline-flex;
+
+    .d-emoji {
+      display: inline-flex;
+    }
```

Comment:
Shouldn't it already be this from your change in emoji.less?

Reply:
This is not needed either, was a leftover of my multiple tries